com.ibm.streamsx.inet/com.ibm.streamsx.inet.ftp/FTPCommand/FTPCommand.xml

```diff
@@ -199,6 +246,26 @@ This optionally parameter controls when the closing of connection after transfer
         <cardinality>1</cardinality>
       </parameter>
       <parameter>
+        <name>connectionTimeout</name>
+        <description>This is the maximum time in seconds that you allow the connection to the server to take. This only limits the connection phase, once it has connected, this option is of no more use. Set to zero to switch to the default built-in connection timeout - 120 seconds. See also the `transferTimeout` parameter.</description>
+        <optional>true</optional>
+        <rewriteAllowed>true</rewriteAllowed>
+        <expressionMode>Expression</expressionMode>
+        <type>int32</type>
+        <cardinality>1</cardinality>
+        <portScope><port>0</port></portScope>
+      </parameter>
+      <parameter>
+        <name>transferTimeout</name>
+        <description>This is the maximum time in seconds that you allow the libcurl transfer operation to take. Normally, name lookups can take a considerable time. Default timeout is 0 (zero) which means it never times out.</description>
+        <optional>true</optional>
+        <rewriteAllowed>true</rewriteAllowed>
+        <expressionMode>Expression</expressionMode>
+        <type>int32</type>
```

Comment:
All other timeouts in operators are float64, can this be consistent with the common practice.